ui/app/templates/components/attributes-section.hbs

```diff
@@ -5,14 +5,11 @@
         {{#if this.prefix}}<span class="is-faded" data-test-prefix>{{this.prefix}}.</span>{{/if}}{{key}}
       </td>
     </tr>
-    <AttributesSection @prefix={{if this.prefix (concat this.prefix "." key) key}} @attributes={{value}} />
+    <AttributesSection @prefix={{if this.prefix (concat this.prefix "." key) key}} @attributes={{value}}
+    @key={{key}} @value={{value}} @editable={{@editable}} @onKVEdit={{@onKVEdit}} @onKVSave={{@onKVSave}} />
```

Comment:
`question`: The component is invoking itself?

Reply:
Yep. This is a nested / tree structure and has been there awhile by the looks of things. For properties that contain a `.` delimiter, it splits them up visually (see items under "connect"):

![image](https://user-images.githubusercontent.com/713991/216718331-cd288388-9d9c-4139-bf36-ff0ca332f0d0.png)

Reply:
Cool!